blog/2023-11-03-netdata-best-practices/index.md

```diff
@@ -0,0 +1,190 @@
+
+The extensive [alerts documentation](https://learn.netdata.cloud/docs/alerting/health-configuration-reference) goes into everything you need to know about configuring alerts.
+
+## Rapid Troubleshooting
+
+Netdata's anomaly detection and metrics correlation features are great tools for rapid troubleshooting. 
+
+1. **Utilize Built-In Anomaly Detection**: Leverage the out-of-the-box anomaly detection for each metric. Since Netdata uses unsupervised machine learning, it does not require labeled data to identify anomalies. This feature can save time and provide immediate insights into unusual system behavior without extensive configuration.
+
+2. **Monitor Anomaly Indicators**: Keep a close eye on the anomaly indicators present on the charts and summary panels. The visual cues—shades of purple—are integral to quickly assessing the severity and presence of anomalies. Adjust your monitoring setup to ensure these indicators are clear and easily noticeable.
+
+3. **Engage with the Anomalies Tab for Deep Dives**: When an anomaly is detected, use the anomalies tab for a granular inspection of the data. This feature is crucial for identifying the start of an anomaly within a specific metric or group of metrics and observing its potential cascading effect through your infrastructure.
+
+4. **Metrics Correlation**: When running Metric Correlations from the Overview tab across multiple nodes, you might find better results if you iterate on the initial results by grouping by node to then filter to nodes of interest and run the Metric Correlations again. Run MC on all nodes. Within the initial results returned group the most interesting chart by node to see if the changes are across all nodes or a subset of nodes. If you see a subset of nodes clearly jump out when you group by node, then filter for just those nodes of interest and run the MC again. This will result in less aggregation needing to be done by Netdata and so should help give clearer results as you interact with the slider. 
+
+5. **Choose the right algorithm**: Use the Volume algorithm for metrics with a lot of gaps (e.g. request latency when there are few requests), otherwise stick with KS2.
+
+To explore all the options to configiure machine learning, you can read more in the [documentation](https://learn.netdata.cloud/docs/ml-and-troubleshooting/machine-learning-ml-powered-anomaly-detection).
+
+## Go forth and optimize!
+
+Ultimately, an optimized Netdata setup is a living system that demands attention and iteration. By fostering an environment where continuous review is part of the culture, your infrastructure will not just keep pace but set the pace in a landscape that is perpetually evolving. 
+
+With Netdata's full suite of features finely tuned to your operational needs, your monitoring will not just inform you—it will empower you.
```

Comment:
```suggestion
With Netdata's full suite of features finely tuned to your operational needs, your monitoring will not just inform you—it will empower you.

You can also start exploring a fully setup environment in one of our Netdata demo rooms [here](https://app.netdata.cloud/spaces/netdata-demo/rooms/all-nodes/overview#metrics_correlation=false&after=-900&before=0&utc=Europe%2FLisbon&offset=%2B0&timezoneName=Dublin%2C%20Lisbon&modal=&modalTab=&modalParams=&selectedIntegrationCategory=deploy.operating-systems&force_play=false&d8a4e0c5-7c79-4145-900e-83a9f06fcb6a--sidebarOpen-bool=true&d8a4e0c5-7c79-4145-900e-83a9f06fcb6a--chartName-val=menu_system).
```